src/Request/ConsoleRequest.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace Bugsnag\Request;
+
+class ConsoleRequest implements RequestInterface
+{
+    /**
+     * The unformated console command.
+     *
+     * @var array
```

Comment:
`string[]` I think is the type here?

Reply:
They're allegedly the same, but `string[]` is more explicit to the expected array elements